### DISCUSSION TITLE
Set available serdes

### DISCUSSION THREAD

**Discussion by woile** [kafbat/kafka-ui#565] [Q&A]
Hi! I was wondering if there's a way to limit which serdes are allowed. I'd like to show our custom serde + the String serde only. Is this possible? I couldn't find it in the docs.
Thanks

**Comment by Haarolean** [comment]
Hey, there's no such a thing atm. I'll convert this into an issue.

**Comment by Haarolean** [comment]
Please follow (and upvote if interested) #627
